lib/services/livesync/livesync-service.ts

```diff
@@ -109,6 +109,9 @@ class LiveSyncService implements ILiveSyncService {
 				this.$hooksService.executeBeforeHooks('watch').wait();
 				this.partialSync(liveSyncData[0].syncWorkingDirectory, watchForChangeActions);
 			}
+			if (this.$options.justlaunch) {
+				process.kill(process.pid);
```

Comment:
This is really strange. What's the idea. Isn't this process.exit? What keeps the process alive?